src/transformState.js

```diff
@@ -5,7 +5,19 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const el of actions) {
+    if (el.type === 'addProperties') {
+      Object.assign(state, el.extraData);
+    } else if (el.type === 'removeProperties') {
+      for (const key of el.keysToRemove) {
+        delete state[key];
+      }
+    } else if (el.type === 'clear') {
+      for (const key in state) {
+        delete state[key];
+      }
+    }
```

Comment:
try to avoid using else if statements, such it decreases readability. It'd be better to apply switch